src/google/protobuf/compiler/csharp/csharp_primitive_field.cc

```diff
@@ -226,7 +226,18 @@ void PrimitiveFieldGenerator::GenerateCloningCode(io::Printer* printer) {
 void PrimitiveFieldGenerator::GenerateCodecCode(io::Printer* printer) {
   printer->Print(
     variables_,
-    "pb::FieldCodec.For$capitalized_type_name$($tag$)");
+    "pb::FieldCodec.For$capitalized_type_name$($tag$, $default_value$)");
```

Comment:
Looks like this PR has several changes related to default value support?

Reply:
The changes made for default values are specifically changes to make default values work with extensions.